libtransmission/platform-quota.c

```diff
@@ -284,7 +284,7 @@ getquota (const char * device)
       spaceused = (int64_t) dq.dqb_curblocks >> 1;
 #elif defined(__APPLE__)
       spaceused = (int64_t) dq.dqb_curbytes;
-#elif defined(__UCLIBC__)
+#elif defined (__UCLIBC__) && !TR_UCLIBC_CHECK_VERSION (1, 0, 17)
```

Comment:
Shouldn't you check for 1.0.18 here? Current condition reads "if uClibc and its version is _not greater or equal to_ (i.e. _less than_) 1.0.17".